src/commands/oas2json.ts

```diff
@@ -61,6 +74,175 @@ const processSchema = (
   })
 }
 
+type ParameterType = {
+  name: string
+  in: string
+  required?: boolean
+  schema?: any
+}
+
+interface OpenApiPath {
```

Comment:
do you need to create your own type? IMO if you look through npm you'll be able to find something like this, but officially maintained by the OAS team

Reply:
hey, can you provide the link please!

Reply:
Literally the second entry if you search for `openapi` on npm: https://www.npmjs.com/package/openapi3-ts